src/components/messageCanvas/messageCanvas.css

```diff
@@ -1,29 +1,39 @@
 .rustic-message-canvas {
-  display: flex;
-  flex-direction: column;
   gap: 4px;
+  position: relative;
 }
 
-.rustic-message-canvas .rustic-message-actions-container {
+.rustic-message-canvas .rustic-message-container {
   padding: 16px;
 }
 
-.rustic-message-canvas .rustic-sender-info {
+.rustic-message-canvas .rustic-message-actions-container {
+  opacity: 0;
+  position: absolute;
+  top: 4px;
+  right: 16px;
   display: flex;
   align-items: center;
-  gap: 8px;
 }
 
-.rustic-message-canvas .rustic-message-footer {
-  opacity: 0;
-  display: flex;
-  align-items: center;
+.rustic-message-canvas .rustic-timestamp {
+  margin-left: 16px;
+  align-self: flex-end;
```

Comment:
It had to do with vertical alignment as it's not completely centred in the design but I'm thinking now it might be too low with flex-end. What do you think?

with `align-self: flex-end;`
<img width="803" alt="Screenshot 2024-05-29 at 3 59 36 PM" src="https://github.com/rustic-ai/ui-components/assets/111031789/428dda13-841e-48bb-b999-5a7b7e470bd2">

without `align-self: flex-end;`
<img width="803" alt="Screenshot 2024-05-29 at 3 59 43 PM" src="https://github.com/rustic-ai/ui-components/assets/111031789/d1289c81-daa7-45f5-aec5-d59900bfdccf">

Reply:
I think we could do without the align-self styling.